autopush/main.py

```diff
@@ -43,6 +46,10 @@ def add_shared_args(parser):
                         dest='config_file', is_config_file=True)
     parser.add_argument('--debug', help='Debug Info.', action="store_true",
                         default=False, env_var="DEBUG")
+    parser.add_argument('--log_level', help='Log level to log', type=str,
+                        default="", env_var="LOG_LEVEL")
+    parser.add_argument('--log_output', help="Log output, stdout or filename",
+                        default="stdout", env_var="LOG_OUTPUT")
```

Comment:
since HUMAN is no longer needed, can probably drop --human_logs (or at least move to obsolete_args() )